### PR TITLE
[cloud-provider-openstack] Clarification of rootDiskSize usage

### PR DESCRIPTION
**Pull request by konstantin-axenov**
## Description

Attempt to make usage of rootDiskSize easier.

## Why do we need it, and what problem does it solve?

There are always difficulties with understanding this part of docs.

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
@konstantin-axenov Just a reminder that PR is stalled

File: ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md [hunk 1]
```diff
@@ -195,3 +195,49 @@ username = {{ nova_service_user_name }}
...
+Если объем диска — ненулевой, то возможны разные ситуации.
+OpenStack-провайдеры могут использовать разные типы томов, для root-диска. Это может быть как сетевой диск, так и локальный диск. Чтобы быть уверенным, лучше уточните этот момент в документации или в технической поддержке провайдера.
+
+Локальные диски имеют свои ограничения, одно из которых то, что ВМ не может мигрировать между гипервизорами. Однако, локальные диски, как правило, дешевле и быстрее сетевых дисков. Поэтому мы рекомендуем следующее:
+* Для master-узла предпочтительнее использовать сетевой диск;
+* Для ephemeral-узла можно использовать локальный диск;
+* Избегайте использования flavor'ов с указанным объемом диска и одновременным использованием параметра `roodDiskSize`. В этом случае cloud-провайдер может взымать плату за использование диска в соответствии с объемом диска, указанным во flavor, но вы не сможете его использовать.
+
+С учетом изложенного выше:
+- Если `rootDiskSize` не установлен, будет использован ephemeral-диск размера, соответствующего указанному во flavor, и типа, указанного в cloud provider'е;
```
Comment (konstantin-axenov) [inline ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md:227]:
```suggestion
- Если `rootDiskSize` не установлен, будет использован ephemeral-диск размера, соответствующего указанному во flavor, и типа, который определён на стороне cloud-provider'а;
```

File: ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md [hunk 2]
```diff
@@ -195,3 +195,49 @@ username = {{ nova_service_user_name }}
...
+- Если `rootDiskSize` указан, будет использован Cinder volume OpenStack (указанного размера с типом тома по умолчанию, указанного в OpenStack). Но, в этом случае есть несколько нюансов:
+  * Для master-узла всегда будет использоваться тип диска, указанный в параметре [volumeTypeMap](cluster_configuration.html#parameters-masternodegroup-volumetypemap) OpenStackClusterConfiguration;
+  * При необходимости, можно [переопределить](#как-переопределить-тип-диска-по-умолчанию-cloud-провайдера) тип диска по умолчанию cloud provider.
```
Comment (konstantin-axenov) [inline ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md:230]:
```suggestion
  * При необходимости, можно [переопределить](#как-переопределить-тип-диска-по-умолчанию-cloud-провайдера) тип диска по умолчанию cloud-provider'a.
```

File: ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md [hunk 3]
```diff
@@ -195,3 +195,49 @@ username = {{ nova_service_user_name }}
...
+
+## Как переопределить тип диска по умолчанию cloud-провайдера?
+
+Если у cloud-провайдера доступно несколько типов, то вы можете указать тип диска по умолчанию для образа, чтобы использовать его при создании ВМ. Для этого необходимо указать название типа диска в метаданных образа.
```
Comment (konstantin-axenov) [inline ee/modules/030-cloud-provider-openstack/docs/FAQ_RU.md:234]:
```suggestion
Если у cloud-провайдера доступно несколько типов, то вы можете указать тип диска по умолчанию для образа. Для этого необходимо указать название типа диска в метаданных образа. Тогда все ВМ, создаваемые из этого образа, будут использовать указанный диск.
```

**Comment by z9r5** [comment]
Cherry-picked into release-1.31

**Pull request merged**